python/lsst/ts/epm/mib_tree_holder.py

```diff
@@ -93,58 +102,110 @@ def _add_default_elements(self) -> None:
             parent=private,
             type=MibTreeElementType.BRANCH,
         )
-        eaton = MibTreeElement(
-            name="eaton",
-            description="eaton",
-            oid="1.3.6.1.4.1.534",
-            parent=enterprises,
-            type=MibTreeElementType.BRANCH,
-        )
-        xups = MibTreeElement(
-            name="xups",
-            description="xups",
-            oid="1.3.6.1.4.1.534.1",
-            parent=eaton,
-            type=MibTreeElementType.BRANCH,
-        )
         self.mib_tree[snmp.name] = snmp
         self.mib_tree[system.name] = system
         self.mib_tree[sys_descr.name] = sys_descr
         self.mib_tree[private.name] = private
         self.mib_tree[enterprises.name] = enterprises
-        self.mib_tree[eaton.name] = eaton
-        self.mib_tree[xups.name] = xups
 
     def _add_mib_elements(self) -> None:
         """Loop over the MIB files and add their contents as a tree
         structure."""
-        for filename in DATA_DIR.glob("*.mib"):
+        for filename in sorted(DATA_DIR.glob("*.mib")):
+            self.log.debug(f"Processing {filename}.")
             with open(filename) as f:
                 lines = f.readlines()
 
             for self._line_num in range(len(lines)):
                 line = lines[self._line_num].strip()
+                mod_id_match = re.match(MODULE_IDENTITY, line)
                 obj_id_match = re.match(OBJECT_IDENTIFIER, line)
                 obj_type_match = re.match(OBJECT_TYPE, line)
-                if obj_id_match:
-                    parent_name = obj_id_match.group(2)
-                    parent: MibTreeElement | None = self._get_parent(parent_name)
-                    assert parent is not None
-                    name = obj_id_match.group(1)
-                    if name == "synaccess":
-                        name = "pdu"
-                    if name == "schneiderElectric":
-                        name = "scheiderPm5xxx"
-                    self.mib_tree[name] = MibTreeElement(
-                        name=name,
-                        description=obj_id_match.group(1),
-                        oid=f"{parent.oid}.{obj_id_match.group(3)}",
-                        parent=parent,
-                        type=MibTreeElementType.BRANCH,
-                    )
+                if mod_id_match:
+                    self._process_mod_id(lines, line, mod_id_match)
+                elif obj_id_match:
+                    self._process_obj_id(obj_id_match)
                 elif obj_type_match:
                     self._process_obj_type(lines, line, obj_type_match)
```

Comment:
I think you could consider to add a `else` condition to throw out an error if nothing matches.